jbpm-workitems/jbpm-workitems-rest/src/main/java/org/jbpm/process/workitem/rest/RESTWorkItemHandler.java

```diff
@@ -133,6 +141,11 @@
         } catch (ClassNotFoundException e) {
             HTTP_CLIENT_API_43 = false;
         }
+
+        // setup pooling connection manager
+        connectionManager = new PoolingHttpClientConnectionManager();
+        connectionManager.setMaxTotal(500);
+        connectionManager.setDefaultMaxPerRoute(50);
```

Comment:
would make sense to have this configurable though it would mean yet another (or more) constructors...